packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/rest_api/FlutterRestInputs.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_api.rest_api
+
+import com.amazonaws.amplify.amplify_api.FlutterApiErrorMessage
+import com.amazonaws.amplify.amplify_api.FlutterApiErrorUtils
+import com.amplifyframework.api.rest.RestOptions
+import java.lang.ClassCastException
+import io.flutter.plugin.common.MethodChannel.Result
+
+
+data class FlutterRestInputs(val flutterInputsMap: Map<String, *>) {
```

Comment:
suggestion: s/FlutterRestInputs/AmplifyAPIRestInputs/

Reply:
I've removed this class and merged it with FlutterApiRequest